sky/skylet/job_lib.py

```diff
@@ -1008,7 +986,7 @@ def cancel_jobs(cls,
             # Print cancelled IDs. Caller should parse by decoding.
             'print(cancelled, flush=True)',
         ]
-        # TODO(zhwu): Backward compatibility, remove after 0.9.0.
+        # TODO(zhwu): Backward compatibility, remove after 0.10.0.
```

Comment:
I am not sure why this was set for 0.9.0, because this code hasn't even entered a stable release yet. @Michaelvll Is 0.10.0 ok for this?

Reply:
offline: we will change this to 0.12.0